R/driver-db2.R

```diff
@@ -29,3 +30,44 @@ setMethod("sqlCreateTable", "DB2/AIX64",
     ))
   }
 )
+
+
+#' @rdname DBI-methods
+# In addition to the `ODBC` API call to `SQLTables`, also
+# attempt to query SYSIBMADM.ADMINTEMPTABLES for any tables
+# that match the arguments.
+#' @usage NULL
+setMethod("odbcConnectionTables", c("DB2/AIX64", "character"),
+  function(conn,
+           name,
+           catalog_name = NULL,
+           schema_name = NULL,
+           table_type = NULL,
+           exact = FALSE) {
+
+    res <- callNextMethod()
+    dfTempTables <- NULL
+    if ( (is.null(schema_name) || schema_name == "SESSION") &&
+         (is.null(catalog_name) || tolower(catalog_name) == conn@info$username) ) {
+      # Also look for temp tables in this case
+      qTable <- getSelector("TABNAME", name, exact)
+      qSchema <- getSelector("TABSCHEMA", schema_name, exact)
+      query <- paste0(
```

Comment:
I think you should include a link to the docs in a comment here.

Reply:
Done / thanks.